### ISSUE TITLE
Error: Couldn't add parts to shopping cart

### ISSUE THREAD

**Issue by philips** [BrickTwo/BrickHunter#214]
Yesterday the plugin was working great for me. Today when I try to transfer items I get "Error: Couldn't add parts to shopping cart".

Do you have any advice on how I can debug? I am a software engineer but not familiar with plugins.

A few http errors I see in the console when BrickHunter fails.
```
pick-a-brick:598 
       POST https://lego.report-uri.com/r/t/csp/wizard 429 (Too Many Requests)

        
       GET https://www.lego.com/cdn/product-assets/element.img.photoreal.192x192/6475205.jpg 404 (Not Found)
pick-a-brick:598 
        
        
       GET https://www.lego.com/cdn/product-assets/element.img.photoreal.192x192/6486299.jpg 404 (Not Found)
6486299.jpg:1 
        
        
       GET https://www.lego.com/cdn/product-assets/element.img.photoreal.192x192/6486299.jpg 404 (Not Found)
Image (async)
```

**Comment by philips** [comment]
I tried in both Chrome and Firefox. Issue reproduces in both.

**Comment by philips** [comment]
Oh, and version 2.4.1

**Comment by BrickTwo** [comment]
Thx for reporting. Looks like LEGO has changed the GraphQL endpoint for adding parts the second time this week. I should have he time to fix this today evening or latest tomorrow.

**Comment by philips** [comment]
Thank you for the quick reply and in advance for the fix. You are doing a great service to the community with this tool.

**Comment by BrickTwo** [comment]
Error has been fixed in version 2.4.2.
Available for firefox, for chrome/edge it should be approved by google in the next 24h.

**Comment by philips** [comment]
Thank you!Sent from a pocket terminal On Jul 28, 2024, at 3:07 AM, BrickTwo ***@***.***> wrote:﻿
Error has been fixed in version 2.4.2.
Available for firefox, for chrome/edge it should be approved by google in the next 24h.
